index.js

```diff
@@ -21,6 +21,12 @@ const exampleProducts = [
 function getCartTotal(cart) {
   let result = 0;
   for (let product of cart) {
+    if(!cart.length){
```

Comment:
You're failing this test due to scope. If your array is empty it will not enter the for loop